src/legacy/ui/public/vis/editors/default/components/default_editor_agg.test.tsx

```diff
@@ -0,0 +1,243 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import { VisState } from 'ui/vis';
+import { AggGroupNames } from '../agg_groups';
+import { DefaultEditorAgg, DefaultEditorAggProps } from './default_editor_agg';
+
+jest.mock('./default_editor_agg_params', () => ({
+  DefaultEditorAggParams: () => null,
+}));
+
+describe('DefaultEditorAgg component', () => {
+  let defaultProps: DefaultEditorAggProps;
+  let onAggParamsChange: jest.Mock;
+  let setTouched: jest.Mock;
+  let onToggleEnableAgg: jest.Mock;
+  let removeAgg: jest.Mock;
+
+  beforeEach(() => {
+    onAggParamsChange = jest.fn();
+    setTouched = jest.fn();
+    onToggleEnableAgg = jest.fn();
+    removeAgg = jest.fn();
+
+    defaultProps = {
+      agg: {
+        id: 1,
+        brandNew: true,
+        getIndexPattern: () => ({}),
+        schema: {},
+        title: 'Metrics',
+        params: {},
+      },
+      aggIndex: 0,
+      aggIsTooLow: false,
+      dragHandleProps: null,
+      formIsTouched: false,
+      groupName: AggGroupNames.Metrics,
+      isDraggable: false,
+      isLastBucket: false,
+      isRemovable: false,
+      metricAggs: [],
+      state: {} as VisState,
+      onAggParamsChange,
+      onAggTypeChange: () => {},
+      setValidity: () => {},
+      setTouched,
+      onToggleEnableAgg,
+      removeAgg,
+    };
+  });
+
+  it('should init with the default set of props', () => {
+    const comp = shallow(<DefaultEditorAgg {...defaultProps} />);
+
+    expect(comp).toMatchSnapshot();
+  });
+
+  it('should open accordion initially', () => {
+    const comp = shallow(<DefaultEditorAgg {...defaultProps} />);
+
+    expect(comp.props()).toHaveProperty('initialIsOpen', true);
+  });
+
+  it('should not show description when no type', () => {
```

Comment:
This will not show the description because of editor is opened by default (not because of the absence of agg type)

Reply:
You're right. Thanks for noticing.
Fixed.